docs/crd.md

```diff
@@ -0,0 +1,60 @@
+# Restoring Deleted CRs
+
+When a Rook CR is deleted, the Rook operator will respond to the deletion event to attempt to clean up the cluster resources. If any data is still present in the cluster, Rook will refuse to delete the CR to ensure data is not lost. The operator will refuse to remove the finalizer on the CR until the underlying data is deleted.
+
+While the underlying Ceph data and daemons continue to be available, the CRs will be stuck indefinitely in a Deleting state in which the operator will not continue to ensure cluster health. Upgrades will be blocked, further updates to the CRs are prevented, and so on. Since Kubernetes does not allow undeleting resources, the command below will allow repairing the CRs without even necessarily suffering cluster downtime.
+
+## Restore Command
+
+The `restore-deleted` command has one required and one optional parameter:
+
+- `<CRD>`: the CRD type that is to be restored, such as CephCluster, CephFilesystem, CephBlockPool and so on.
+- `[CRName]`: is the name of the specific CR which you want to restore since there can be multiple instances under the same CRD. For example, if there are multiple CephFilesystems stuck in deleting state, a specific filesystem can be restored: `restore-deleted cephfilesystem filesystem-2`.
+
+```bash
+kubectl rook-ceph restore-deleted <CRD> [CRName]
+```
+
+## CephCluster Restore Example
+
+```bash
+kubectl rook-ceph restore-deleted cephcluster [cephClusterName]
+
+Info: Detecting which resources to restore for crd "cephcluster"
+Info: Restoring CR my-cluster
+Warning: The resource my-cluster was found deleted. Do you want to restore it? yes | no
+
+yes
+Info: proceeding with restoring deleting cr
+Info: Scaling down the operator to 0
+Info: Backing up kubernetes and crd resources
+Info: Backed up crd cephcluster/my-cluster in file cephcluster-my-cluster.yaml
+Info: Deleting validating webhook rook-ceph-webhook if present
+Info: Fetching the UID for cephcluster/my-cluster
+Info: Successfully fetched uid 9575a4c9-7d0f-4b98-93d5-0b7dab7efeb4 from cephcluster/my-cluster
+Info: Removing ownerreferences from resources with matching uid 9575a4c9-7d0f-4b98-93d5-0b7dab7efeb4
+Info: Removing owner references for secret cluster-peer-token-my-cluster
+Info: Removed ownerReference for Secret: cluster-peer-token-my-cluster
+
+Info: Removing owner references for secret rook-ceph-admin-keyring
+Info: Removed ownerReference for Secret: rook-ceph-admin-keyring
+---
```

Comment:
If these are to show removed text, I'd suggest changing to dots and only one line of them is needed. But perhaps for this example we don't need to remove any text since it's not terribly long for this command.
```suggestion
...
```